datalad/downloaders/tests/test_http.py

```diff
@@ -123,26 +126,46 @@ def enter_new(self):
         # pop last used credential, so we would use "new" ones
         del self._fixed_credentials[0]
 
+
```

Comment:
no -- it is already in its mother branch into which I have merged that
nf-ui-auth-downloaders... just fire up gitk --all and I bet you will see
it

search for skip  if you don't
-- 
Yaroslav O. Halchenko
Center for Open Neuroscience     http://centerforopenneuroscience.org
Dartmouth College, 419 Moore Hall, Hinman Box 6207, Hanover, NH 03755
Phone: +1 (603) 646-9834                       Fax: +1 (603) 646-1419
WWW:   http://www.linkedin.com/in/yarik